includes/admin/class-wp-job-manager-addons-landing-page.php

```diff
@@ -60,16 +61,16 @@ public function register_styles() {
 		WP_Job_Manager::register_style( 'job_manager_admin_landing_css', 'css/admin-landing.css', [ 'job_manager_brand' ] );
 	}
 
+
 	/**
-	 * Add add-on menu items if needed.
+	 * Add Applications add-on menu item if needed.
 	 *
 	 * @access private
 	 * @since $$next-version$$
 	 */
-	public function add_menu_items() {
-
-		$parent_page = 'edit.php?post_type=job_listing';
-		$badge_text  = __( 'Pro', 'wp-job-manager' );
+	public function add_applications_menu_item() {
+		$was_addon_installed = ! ! get_option( 'wp_job_manager_applications_version', false );
```

Comment:
Nothing wrong with how it is, I 've seen it tackled like this more often:
```suggestion
		$was_addon_installed = (bool) get_option( 'wp_job_manager_applications_version', false );
```

Reply:
Yeah that's probably more readable.